src/formats/pnm.c

```diff
@@ -0,0 +1,222 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Please format the code according to .clang-format.

Reply:
Sorry, totally missed it! Will do once I make the larger structural changes above.